specification/network/resource-manager/Microsoft.Network/stable/2018-06-01/virtualWan.json

```diff
@@ -0,0 +1,2105 @@
+        },
+        "description": "Deletes a VirtualWAN.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The resource group name of the VirtualWan."
+          },
+          {
+            "name": "VirtualWANName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the VirtualWAN being deleted."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Request successful. VirtualWAN deleted."
+          },
+          "202": {
+            "description": "Request received successfully. VirtualWAN deletion is in progress; follow the Location header to poll for final outcome."
+          },
+          "204": {
+            "description": "No VirtualWANs exist by the name provided."
+          },
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "./network.json#/definitions/Error"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/virtualWans": {
+      "get": {
+        "operationId": "VirtualWANs_ListByResourceGroup",
+        "x-ms-examples": {
+          "VirtualWANListByResourceGroup": { "$ref": "./examples/VirtualWANListByResourceGroup.json" }
+        },
+        "description": "Lists all the VirtualWANs in a resource group.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The resource group name of the VirtualWan."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Request successful. Returns the details of all the VirtualWANs in the resource group.",
+            "schema": {
+              "$ref": "#/definitions/ListVirtualWANsResult"
+            }
+          },
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "./network.json#/definitions/Error"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Network/virtualWans": {
+      "get": {
+        "operationId": "VirtualWANs_ListBySubscriptionId",
```

Comment:
@jianghaolu, is it common naming for now? As I remember previously it was ListByResourceGroup+List pair. Was this changed recently?

Reply:
No this should be `VirtualWANs_List". Good catch!